src/main/java/org/terasology/polyworld/raster/RiverRasterizer.java

```diff
@@ -54,7 +54,7 @@
     @Override
     public void initialize() {
         BlockManager blockManager = CoreRegistry.get(BlockManager.class);
-        water = blockManager.getBlock("core:water");
+        water = blockManager.getBlock("CoreBlocks:water");
```

Comment:
Use same capitalization as in other changes: 
```suggestion
        water = blockManager.getBlock("CoreBlocks:Water");
```